sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -387,6 +389,82 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
     }
   }
 
+  test("filter pushdown - timestamp(TIMESTAMP_MILLIS)") {
```

Comment:
I think we should also test INT96 timestamp type. Also maybe when `PARQUET_FILTER_PUSHDOWN_TIMESTAMP_ENABLED` is disabled.